uktrade/data-hub-frontend
# Feature/cls2 430 view task detail

## Description of change

Users need to be able to view full details relating to a task.

This PR depends on the API branch feature/CLS2-462-tasks-API-should-return-details-for-investment-project-task

## Test instructions

Create an Investment Project Task and get the id of the related Task. View the Investment project task on http://localhost:3000/tasks/[taskId]. Make sure it's the id of the related Task and not the Investment Project Task (see steps below)

- [Create a Investment Project Task in the API.](http://localhost:8000/admin/task/investmentprojecttask/add/)
  - Create a Task for the Investment Project Task ![image](https://github.com/uktrade/data-hub-frontend/assets/699259/35c839f2-9b99-400b-b1a9-ad6b0b75f23f) 
     - Only the fields from Title onwards need completing <img width="785" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/699259/871ff0bb-3d99-4dd4-84ad-7e18670079c0">
  -  Either create or select an Investment Project.
- Get the id of the task by editing the related task <img width="576" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/699259/6a1bf966-ae3a-466b-af63-95f704d6349f">

Visit: http://localhost:3000/tasks/[taskId] to view the details of the Investment Project Task on the front end.

## Screenshots

### Before

n/a

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/79982dfa-6571-47b0-a01c-11a1b8d3cd67)
## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
